server/lib/handlebars.js

```diff
@@ -190,7 +190,7 @@ handlebars.registerHelper('formatOrderAmountWithInterval', order => {
   }
 
   const formattedAmount = formatCurrency(order.totalAmount, order.currency);
-  const subscription = order.subscription;
+  const subscription = order.Subscription;
```

Comment:
To check

Reply:
Comes from https://github.com/opencollective/opencollective-api/blob/ef2c3845349886048f181b4622404f965a2cf134/server/models/Collective.ts#L1821-L1823